### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by Alwaysin**
English and french documentation are not equal, we should at least get them on par and then think aout adding the missing bits. I'll try to do my best in the coming days/weeks in this PR. Help is welcome, I allowed edit from maintainers.

Sorry if it's a bit unorganized, I'm not used to use Github, PRs and so on. At least now I know how to make several commits into the same PR :D

For the moment, I think we'll be able to close: https://github.com/FreshRSS/FreshRSS/issues/785

### CODE REVIEW

File: docs/fr/users/05_Configuration.md [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+| it       | 86.4%      |
+| kr       | 96.3%      |
+| nl       | 95.4%      |
+| pt-br    | 87.4%      |
+| ru       | 36.4%      |
+| tr       | 88.1%      |
+| zh-cn    | 99.0%      |
+
 ##Thème
 Les goûts et les couleurs, ça ne se discute pas. C'est pourquoi FreshRSS propose six thèmes officiels :
```
Comment (Alkarex) [inline docs/fr/users/05_Configuration.md]: Huit thèmes :-)

File: docs/fr/users/05_Configuration.md [hunk 2]
```diff
@@ -1,10 +1,28 @@
 # Personnaliser la vue
 
 ##Langue
-À l'heure actuelle, FreshRSS est disponible en français et en anglais. Après validation de ce choix, la totalité de l'interface sera affichée dans la langue choisie.
+À l'heure actuelle, FreshRSS est disponible en 13 langues. Après validation de ce choix, l'interface sera affichée dans la langue choisie, même si certaines parties de l'interface peuvent ne pas encore avoir été traduites. Si vous voulez aider à la traduction, regardez comment vous pouvez [contribuer au projet](../contributing.md#contribute-to-internationalization-i18n).
 
 Il y a des parties de FreshRSS qui ne sont pas traduites et qui n'ont pas vocation à l'être. Pour le moment, les logs visibles dans l'application ainsi que celle générées par le script de mise à jour automatique en font partie.
 
+Not all languages are equals regarding completion:
```
Comment (Alkarex) [inline docs/fr/users/05_Configuration.md]: Mettre en français, ou peut-être supprimer ou raccourcir cette section

**Comment by Alkarex** [comment]
Thanks for this first part already @Alwaysin 👍

File: docs/fr/users/05_Configuration.md (hunk 2 above)
Comment (Alwaysin) [reply docs/fr/users/05_Configuration.md]: Effectivement, j'ai supprimé cette histoire de pourcentage qui ne sert à rien au final.

**Comment by Alkarex** [comment]
Thanks. It is already a good improvement. We can merge when you want.

**Review by Alkarex — APPROVED** [review]

**Comment by Alwaysin** [comment]
I'd like to get a bit ore done before merging. Do you think gifs would be a good idea instead of having lots of text? For example for "04_Subscriptions.md - Adding a feed" we could have:

**Comment by Alkarex** [comment]
@Alwaysin Ok, and yes, Looks good to me 👍

**Comment by Frenzie** [comment]
Interestingly, I just found this asset branch approach: https://stackoverflow.com/a/42677655/2470572

https://gist.github.com/joncardasis/e6494afd538a400722545163eb2e1fa5

I haven't researched whether it's a good idea, but I suppose it should do the job of not having to pull in many megabytes just for playing with some text files unless you explicitly fetch the assets branch.

I'm inclined to think a completely separate assets repository would be a safer approach for having a proper commit history and all that.

**Comment by Alwaysin** [comment]
Which solution do I use then? :)

**Comment by Alkarex** [comment]
@Alwaysin I am a bit in doubt of the best approach. I think I can best like the idea of making another repository for those large assets. Pros / cons welcome.

**Comment by Alkarex** [comment]
I will create a repository for the GIFs and related assets tomorrow. Pros / cons compared to other solutions welcome in the meantime, and if needed, we can always change from that approach to something else in the future.

**Comment by Alkarex** [comment]
Sorry for the delay. I suggest we use https://github.com/FreshRSS/documentation for the documentation assets that are a bit large.
@Alwaysin feel free to make a PR to https://github.com/FreshRSS/documentation with your GIFs - trying to keep the size reasonable 😉

**Comment by Alkarex** [comment]
I merge to get the existing improvements in the soon-to-be-released FreshRSS 1.13, but please do open another PR when you have more substance :-)

**Pull request merged**